<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Update Profile" time="4" tests="0" failures="0" errors="2">
   <testsuite name="Update Profile" tests="0" failures="0" errors="2" time="4" timestamp="2017-10-12 13:44:00" hostname="Gituto Nyawira - Gituto-PC" id="Test Suites/Profile/Update Profile">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\Gituto Nyawira\\git\\VSMS-Test-Automation\\Reports\\Profile\\Update Profile\\20171012_134353"/>
         <property name="logFiles" value="C:\\Users\\Gituto Nyawira\\git\\VSMS-Test-Automation\\Reports\\Profile\\Update Profile\\20171012_134353\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="Gituto Nyawira - Gituto-PC"/>
         <property name="hostAddress" value="192.168.43.189"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="4.8.0.2"/>
      </properties>
      <testcase name="Test Cases/Profile/Update Profile - Iteration 1" classname="Test Cases/Profile/Update Profile" status="ERROR">
         <error type="ERROR" message="Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN'"/>
         <system-out><![CDATA[2017-10-12 13:44:01 - [TEST_CASE][ERROR] - Test Cases/Profile/Update Profile - Iteration 1: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN'

2017-10-12 13:44:03 - [TEST_STEP][PASSED] - callTestCase: null]]></system-out>
         <system-err><![CDATA[2017-10-12 13:44:01 - [TEST_CASE][ERROR] - Test Cases/Profile/Update Profile - Iteration 1: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN']]></system-err>
      </testcase>
      <testcase name="Test Cases/Profile/Update Profile - Iteration 2" classname="Test Cases/Profile/Update Profile" status="ERROR">
         <error type="ERROR" message="Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN'"/>
         <system-out><![CDATA[2017-10-12 13:44:03 - [TEST_CASE][ERROR] - Test Cases/Profile/Update Profile - Iteration 2: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN'

2017-10-12 13:44:03 - [TEST_STEP][PASSED] - callTestCase: null]]></system-out>
         <system-err><![CDATA[2017-10-12 13:44:03 - [TEST_CASE][ERROR] - Test Cases/Profile/Update Profile - Iteration 2: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN']]></system-err>
      </testcase>
      <system-out><![CDATA[2017-10-12 13:44:00 - [TEST_SUITE][ERROR] - Update Profile: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN']]></system-out>
      <system-err><![CDATA[2017-10-12 13:44:00 - [TEST_SUITE][ERROR] - Update Profile: Test Cases/Profile/Update Profile FAILED because (of) java.lang.IllegalArgumentException: Cannot find test case with id 'Test Cases/LOGIN']]></system-err>
   </testsuite>
</testsuites>
